index.html

```diff
@@ -3,9 +3,61 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style/index.css">
     <title>Document</title>
 </head>
 <body>
-    
+    <H1>Casey's Shop</H1>
+    <Section class='top-nav'>
+        <h3>Casey Boyland</h3>
+            <nav>
+                <a href="http://index.html">Home</a>
+                <a href="http://about.html">About</a>
+                <a href="http://contact.html">Contact</a>
+                <a href="http://projects.html">Projects</a>
+            </nav>
+    </Section>
+
+    <Section class='my-name'>
+        <p>Casey Boyland</p>
+        <div class='my-name-link'>
+        <a href="http://projects.html">Projects</a>
+        </div>
+    </Section>
+
+    <section class='gallery'>
+        <div class='cutting-board'>
+        <img src='https://www.learningtosmoke.com/wp-content/uploads/2017/03/IMG_0447-819x1024.jpg'>
+        <p>Cutting Board</p>
+        </div>
+        <div class='coasters'>
+        <img src='https://5.imimg.com/data5/GJ/EB/TE/SELLER-2063983/wooden-coasters-500x500.jpg'>
+        <p>Coasters</p>
+        </div>
+        <div class='bottle-openers'>
+        <img src='https://cdn.shopify.com/s/files/1/1395/9647/products/Bulk_Wood_Bottle_Openers_Custom_Handmade_Logo_grande_8398c588-fe50-44ed-ae9e-37a223cc8c73_grande.jpg?v=1544238876'> 
+        <p>Bottle Openers</p>
+        </div>
+    </section>
+
+    <section class='gallery-2'>
+        <div class='wall-art'>
+        <img src='https://pixl.varagesale.com/http://s3.amazonaws.com/hopshop-image-store-production/153681384/ed289cc0bb37468ac4b331dfbab2345b.jpg?_ver=large_uploader_thumbnail&w=640&h=640&fit=crop&s=c5e05e014a6ba768dd19dabc8e967aa4'>
+        <p>Wall Art</p>
+        </div>
+        <div class='coffee-table'>
+        <img src='https://i0.wp.com/sellmesomething.co.uk/wp-content/uploads/2019/11/Rustic-Vintage-Industrial-Solid-Wood-Coffee-Table.jpg?fit=1500%2C1500&ssl=1'>
+        <p>Custom Furniture</p>
+        </div>
+        <div class='desk'>
+        <img src='https://cb2.scene7.com/is/image/CB2/DrommenDeskSHS18_1x1'>
+        <p>Desks</p>
+        </div>
+    </section>
+
+    <footer>
+        <a href="http://contact.html">Contact Me</a>
```

Comment:
awesome job linking to a contact page!

Reply:
If you wanted to, you could add your social media somewhere as well !

Reply:
Thank you so much for the feedback Brandy! I appreciate you helping me out with everything.